src/System.Runtime.Extensions/src/System/IO/TextWriter.cs

```diff
@@ -21,7 +21,17 @@ public abstract partial class TextWriter : MarshalByRefObject, IDisposable
     {
         public static readonly TextWriter Null = new NullTextWriter();
 
-        protected char[] CoreNewLine = Environment.NewLine.ToCharArray();
+        // We don't want to allocate on every TextWriter creation, so cache the char array.  
+        private static readonly char[] CoreNewLineStatic = Environment.NewLine.ToCharArray();
```

Comment:
Or, just `s_coreNewLine` (without the `Static` suffix since the `s_` prefix implies "Static")